File: core/src/main/java/y2k/joyreactor/TagPost.java
package y2k.joyreactor;

import java.io.Serializable;

/**
 * Created by y2k on 11/24/15.
 */
public class TagPost implements Serializable {

    public int tagId;
    public int postId;

    public TagPost(int tagId, int postId) {
        this.tagId = tagId;
        this.postId = postId;
    }
}

File: core/src/main/kotlin/y2k/joyreactor/TagPost.kt
package y2k.joyreactor

import y2k.joyreactor.services.repository.DataSet
import java.io.Serializable

/**
 * Created by y2k on 11/24/15.
 */
class TagPost(val tagId: Long, val postId: Long) : Serializable, DataSet.Dto {

    override var id: Long = 0
}